### PR TITLE
adding fb sharing method

### PR DESCRIPTION
**Pull request by drakerehfeld**

### CODE REVIEW

**Comment by drakerehfeld** [comment]
this is a work in progress -- do not merge!

**Comment by drakerehfeld** [comment]
- [ ] edit models to make publishable
- [ ] allow get request of user id for other viewers
- [ ] facebook app integration

File: client/templates/home/home.js [hunk 1]
```diff
@@ -83,20 +83,29 @@ Template.Home.helpers({
...
 		//get the artist arrays
 		var artistsShort = getTop5ArtistsShort();
 		var artistsMedium = getTop5ArtistsMedium();
 		var artistsLong = getTop5ArtistsLong();
-		var stringArtists;
+		var stringArtists = "";
 		
 		//loop through 1-4
-		for (var i = 0; i < 4; i++){
-			stringArtists = stringArtists + artistsShort[i] + ", " artistsMedium[i] + ", " artistsLong[i] + ", ";
```
Comment (drakerehfeld) [inline client/templates/home/home.js]: lol when you think you're writing in python for a hot sec

**Comment by alexpareto** [comment]
bitmoji
@drakerehfeld how's this coming along?

**Comment by drakerehfeld** [comment]
Literally hate you. 

> On Sep 14, 2016, at 18:18, Alex Pareto <notifications@github.com> wrote:
> 
> 
> @drakerehfeld how's this coming along?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 

-- 
Notice: This message, and any attachments, contain(s) information that may 
be confidential or protected by -privilege from disclosure and is intended 
only for the individual or entity named above. No one else may disclose, 
copy, distribute or use the contents of this message for any purpose. Its 
unauthorized use, dissemination or duplication is strictly prohibited and 
may be unlawful. If you receive this message in error or you otherwise are not 
an authorized recipient, please immediately delete the message and any 
attachments and notify the sender.

**Comment by drakerehfeld** [comment]
it needs lots of work so we can just table this lol

**Pull request closed without merging**